src/Card/CardDivider.tsx

```diff
@@ -20,4 +20,4 @@ const styles = StyleSheet.create({
   },
 });
 
-CardDivider.displayName = 'CardDivider';
+CardDivider.displayName = 'Card.Divider';
```

Comment:
Why not `CardDivider`

Reply:
The name should be the same as the use-case, therefore I have changed it. As users are going to use it as Card.Divider. Also, this will help detect the parent directory.